aemrules.properties

```diff
@@ -29,4 +35,4 @@ defaults.mavenArtifactId=aemrules
 0.8.sqVersions=[5.6.1,6.3]
 0.8.date=2015-08-31
 0.8.changelogUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/tag/v0.8
-0.8.downloadUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/download/v0.8/aemrules-0.8.jar
+0.8.downloadUrl=https://github.com/Cognifide/AEM-Rules-for-SonarQube/releases/download/v0.8/aemrules-0.8.jar
```

Comment:
GH wants you to reinstate the final linebreak here.

Reply:
Fixed.